src/lib/filterTasksByClientClassification.ts

```diff
@@ -0,0 +1,24 @@
+export default function filterTasksByClientClassification(
+  tasks: TaskData[],
+  client: Client
+): TaskData[] {
+  return tasks.filter(
+    ({ classification }): boolean => {
+      return Object.keys(classification).reduce<boolean>(
+        (res, key): boolean => {
+          const classifiers = classification[key];
+          if (classifiers) {
+            return (
+              res &&
+              classifiers.some(
+                (val: string | number): boolean => val === client[key]
+              )
+            );
+          }
+          return res && true;
```

Comment:
Also, if `classifiers` is missing and this line is evaluated, maybe we should add a test case for that. Seems important enough